app/cisco/ccna/network-fundamentals-guide/page.css

```
@@ -0,0 +1,313 @@
+
+.ccna-network-fundamentals-page .mermaid-caption {
+  text-align: center;
+  color: var(--guide-text-dim);
+  font-size: 0.86rem;
+  margin-top: -10px;
+  margin-bottom: 28px;
+}
+
+.ccna-network-fundamentals-page code,
+.ccna-network-fundamentals-page .mono {
+  font-family: var(--font-mono), "SFMono-Regular", Consolas, monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)